portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,39 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```

Comment:
Do you want to use https://github.com/google/google-java-format to format the code (e.g. sort the imports).

Replace the wildcard imports with actual imports in use: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports